tests/unit/DefineCompany.spec.ts

```diff
@@ -77,7 +77,7 @@ describe('Define Company view', () => {
 
   it('displays folio number when it is a premium account', () => {
     store.state.stateModel.nameRequest.entityType = 'BC'
-    sessionStorage.setItem('CURRENT_ACCOUNT', 'PREMIUM')
+    store.state.stateModel.accountInformation.accountType = 'PREMIUM'
```

Comment:
Please use the new account type enum :)

Also below on line 91.

Reply:
Did we not want to use the strings for unit tests to catch changes in the account type?

I thought we did this for other things, but maybe I am thinking something else.

Reply:
Oops, this is a unit test file. You are correct!